mmv1/products/networkconnectivity/ServiceConnectionPolicies.yaml

```diff
@@ -0,0 +1,131 @@
+update_verb: :PATCH
+update_mask: true
+description:
+  'Manage Service Connection Policies.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/secure-web-proxy/docs/reference/networkconnectivity/rest/v1/projects.locations.networkConnectionPolicies'
+  guides:
+    'About Service Connection Policies': 'https://cloud.google.com/vpc/docs/about-service-connection-policies#service-policies'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format:
+  ['projects/{{project}}/locations/{{location}}/serviceConnectionPolicies/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_policy_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-network-connectivity-policy'
+      consumer_network_name: 'consumer-net'
+      consumer_subnetwork_name: 'consumer-subnet'
+      producer_network_name: 'producer-net'
+      producer_subnetwork_name: 'producer-subnet'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of a ServiceConnectionPolicy. Format: projects/{project}/locations/{location}/serviceConnectionPolicies/{service_connection_policy} See: https://google.aip.dev/122#fields-representing-resource-names
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    url_param_only: true
+    description: |
+      The location of the ServiceConnectionPolicy.
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      The timestamp when the resource was created.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      The timestamp when the resource was updated.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'serviceClass'
+    description: |
+      The service class identifier for which this ServiceConnectionPolicy is for. The service class identifier is a unique, symbolic representation of a ServiceClass.
+      It is provided by the Service Producer. Google services have a prefix of gcp. For example, gcp-cloud-sql. 3rd party services do not. For example, test-service-a3dfcx.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      Free-text description of the resource.
+  - !ruby/object:Api::Type::String
+    name: 'network'
```

Comment:
Same here, is it possible to create this resource without specifying a network?

Reply:
No it's not, thanks for that.

Reply:
The other fields look good, but I believe `network` here still needs to be marked as `immutable`.

Reply:
@roaks3 I was testing this scenario, if I add the network as immutable, the test fails because the API requires the network in the payload.

Reply:
Hmm, that's strange. Could you provide any more info on that? I think we should be able to configure this property in a way that works.

Alternatively, you could submit it with the `immutable: true` and then I can see the failure in the logs in VCR.